start_2

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
```

Comment:
a lot of duplicate code from start_1... consider pulling the duplicate code (or at least the functions) into a shared file and source it.